app/vmalert/group.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/VictoriaMetrics/VictoriaMetrics/lib/prompbmarshal"
 )
 
+const GroupIntervalOffSetNotExplicitlySpecified = time.Nanosecond * -1
```

Comment:
I think, this const is not needed.  Just use check if g.IntervalOffset < 0

Reply:
Agree with @f41gh7